ROADMAP.md

```diff
@@ -0,0 +1,183 @@
+# Refactoring
+
+Goals:
```

Comment:
Could we have a goal around testing: integration, end to end, etc.

I think one of the biggest gaps within this library is the suite of testing to provide better confidence to the general public on the reliability and resiliency of the library.